physics/mynnedmf_wrapper.F90

```diff
@@ -475,6 +475,32 @@ SUBROUTINE mynnedmf_wrapper_run(        &
                 qnifa(i,k) = qgrs_ice_aer_num_conc(i,k)
               enddo
             enddo
+          else if(mraerosol) then
+            FLAG_QI = .true.
+            FLAG_QNI= .true.
+            FLAG_QC = .true.
+            FLAG_QNC= .true.
+            FLAG_QNWFA= .false.
+            FLAG_QNIFA= .false.
+            p_qc = 2
+            p_qr = 0
+            p_qi = 2
+            p_qs = 0
+            p_qg = 0
+            p_qnc= 0
+            p_qni= 0
+            do k=1,levs
+              do i=1,im
+                sqv(i,k)   = qgrs_water_vapor(i,k)
+                sqc(i,k)   = qgrs_liquid_cloud(i,k)
+                sqi(i,k)   = qgrs_ice_cloud(i,k)
+                qnc(i,k)   = qgrs_cloud_droplet_num_conc(i,k)
+                qni(i,k)   = qgrs_cloud_ice_num_conc(i,k)
+                ozone(i,k) = qgrs_ozone(i,k)
+                qnwfa(i,k) = 0.
```

Comment:
Similarly to prior comment... This is setting aerosol numbers to zero for mraerosol, but aren't aerosol numbers getting mixed when using the earlier ```ltaerosol``` setting?  I am not understanding these apparent inconsistencies.

Reply:
qnwfa and qnifa are diagnosed from merra2. The vertical mixing is not needed.